package.json

```diff
@@ -28,7 +28,7 @@
   "scripts": {
     "configure": "cmake-js configure",
     "build": "cmake-js compile",
-    "install": "npm run build",
+    "install": "yarn build",
```

Comment:
I would change this to `cmake-js compile` which should run perfectly with npm and yarn.